packages/react-ui/src/NakedList/index.js

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+
+export const NakedList = styled.ul`
```

Comment:
I propose to automagically wrap children with some StyledLi 
```
React.Children.toArray(children).map( node => <StyledLi>node</StyledLi> )
```

Reply:
Je ne suis vraiment pas fan de l'idée de wrapper dans un <li> automatiquement, ça implique un savoir supplémentaire quand tu implémentes la liste, ça complexifie le code et ça crée des corner cases: qu'est-ce qu'il se passe si t'as text + un element + texte, ça fait 3 `<li>` ? Et est-ce que les `<>` sont pris en compte etc. ?

Reply:
Ca complique aussi un peu le refacto si tu veux passer d'une `<FlatList>` à un `<ul>` standard

Reply:
oki, moi c'etait plutot le nesting de style que je tenter d'éviter. mais tu as raison sur le coté magique peut etre fournir un FlatLi sans padding pour éviter d'avoir a rajouter du styled component dans le front

Reply:
et le
```
  & > li {
    padding-left: 0;
  }
```
Ca ne te plait pas ? Ca complique un peu le fait d'ajouter un padding-left au li par la suite (il faudrait le wrapper) mais ça évite d'avoir à faire à un import supplémentaire

Reply:
Oui je comprend que ca complexifie le refacto, ca fait plus de cas remplacer. A toi de voir.
Pour expliquer un peu plus pourquoi je prefere un import de plus par rapport au nesting. 
Ca me parait plus cohérent,  ou tout est un composant donc explicite dans l'utilisation. le nesting crée un lien implicite entre les li enfant de Flatlist un peu comme le fait qu'on rajoute des li, il faut se rappeler que si on veut du padding il faut surcharger le comportement (et comme on utilise du nesting il faut ajouter un poid supplémentaire pour que la regle css l'emporte).